src/third_party/lz4-r131/SConscript

```diff
@@ -0,0 +1,17 @@
+# -*- mode: python -*-
+
+Import("env")
+
+env = env.Clone()
+
+if env.ToolchainIs('clang', 'GCC'):
+    env.Append(CCFLAGS=['-I. -std=c99 -Wall -Wextra -Wundef -Wshadow -Wcast-align -Wcast-qual -Wstrict-prototypes -pedantic -DXXH_NAMESPACE=LZ4_'])
```

Comment:
Please, remove all `-W` related things - these are warnings and should be enabled in top-level Scons. With `-pedantic` is the same. `-I.` also looks excessive, all the files are in the same folder and are using `include ""` style. So the line will be:
`-std=c99 -DXXH_NAMESPACE=LZ4_`